web/crux/src/app/deploy/deploy.mapper.ts

```diff
@@ -254,7 +256,7 @@ export default class DeployMapper {
     return state ? (containerStateToJSON(state).toLowerCase() as ContainerStateEnum) : null
   }
 
-  commonConfigToAgentProto(config: MergedContainerConfigData): CommonContainerConfig {
+  commonConfigToAgentProto(config: MergedContainerConfigData, storage?: Storage): CommonContainerConfig {
```

Comment:
why is it not in the config itself?

Reply:
Because the Storage is the actual storage model from DB with access keys and stuff, so it's not the storage config from the config page.